netclient/ncutils/netclientutils.go

```diff
@@ -128,10 +129,19 @@ func IsEmptyRecord(err error) bool {
 // GetPublicIP - gets public ip
 func GetPublicIP() (string, error) {
 
-	iplist := []string{"https://ip.client.gravitl.com", "https://ifconfig.me", "https://api.ipify.org", "https://ipinfo.io/ip"}
+	iplist := []string{"https://ip.server.gravitl.com", "https://ifconfig.me", "https://api.ipify.org", "https://ipinfo.io/ip"}
+
+	for network, ipService := range global_settings.PublicIPServices {
+		logger.Log(3, "User provided public IP service defined for network", network, "is", ipService)
```

Comment:
here you say User provided IP service defined for network... however, when you add entries to this map the key appears to be a server, see functions/daemon.go line 102. this would yield api.nm.mydomain.com as the key and not network names